projects/igniteui-angular/src/lib/combo/combo.component.html

```diff
@@ -1,4 +1,4 @@
-<igx-input-group #inputGroup [type]="type" (click)="onClick($event)">
+<igx-input-group #inputGroup [type]="type === 'search' ? 'box' : type" (click)="onClick($event)">
```

Comment:
Remind me, why have you decided to go with box here and line in the select component?

Reply:
<img width="894" alt="Screenshot 2025-01-28 at 10 44 02" src="https://github.com/user-attachments/assets/40739264-7f8e-44d4-98bd-7c5184fa8977" />

It's set as default value for the component